### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by RohanDudeja**

### CODE REVIEW

**Comment by ShankarParimi** [comment]
Every PR should have right title of the PR and Description about the changes. Make a practice of it.

File: day1/exercise1/solution.go [hunk 1]
```diff
@@ -0,0 +1,56 @@
+package main
+
+import "fmt"
+
+type matrix struct {
+	row, col int
```
Comment (ShankarParimi) [inline day1/exercise1/solution.go]: always create them as separate properties although row and col are int type.

File: day1/exercise1/solution.go [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+	for r := 0 ; r < 2 ; r++ {
+		m.val[r] = make([]int, C)
+		for c := 0; c < C; c++ {
+			m.val[r][c] = r + c
+		}
+	}
+	return &m
+}
+func (m matrix)print() {
+	for r:=0;r<m.getRowCount();r++{
```
Comment (ShankarParimi) [inline day1/exercise1/solution.go]: this doesn't print in json format right?

File: day1/exercise1/solution.go (hunk 1 above)
Comment (ShankarParimi) [reply day1/exercise1/solution.go]: type matrix struct {
	row int
        col int
        val [] [] int
       }

Comment (RohanDudeja) [reply day1/exercise1/solution.go]: cool

**Pull request merged**